recipe/meta.yaml

```diff
@@ -23,27 +23,34 @@ requirements:
     - setuptools
     - wheel
     - cython
-    - fftw
     - {{ pin_compatible('numpy') }}
+    - fftw 3.3.9
   run:
+    - python
     - _openmp_mutex  # [linux]
     - llvm-openmp    # [osx]
-    - fftw
+    - fftw >=3
     - numpy
-    - python
     - scikit-learn >=0.20
     - scipy
+    - pynndescent >=0.5.0,<0.6.0
```

Comment:
Maybe move this and `fftw` to `run_constrained`?

Reply:
`fftw` is needed at runtime, but I've moved `pynndescent` to `run_constrained`. Thanks!